internal/collector/nginxossreceiver/README.md

```diff
@@ -33,10 +33,10 @@ Example:
 
 ```yaml
 receivers:
-  nginx:
+  nginx_receivers:
```

Comment:
The names here are correct and don't need to be updated.
In the agent conf they are called `nginx_receivers` but when converted to otel configuration it is called `nginx`.
This README is for documenting the otel config for the receiver not the agent config if that makes sense.

Reply:
I see thanks!